lib/utils.js

```diff
@@ -20,7 +20,6 @@ require('markdown-link', 'mdlink');
 require('minimist');
 require('mixin-deep', 'merge');
 require('object.pick', 'pick');
-require('remarkable', 'Remarkable');
```

Comment:
When continuing to use `lazy-cache` I got a `Remarkable` is not a constructor error. I'm not sure that there is much value in `lazy-cache` for `remarkable` when it's doing the majority of the heavy lifting for this module.